docs/user/spark/pyspark.rst

```diff
@@ -0,0 +1,91 @@
+GeoMesa PySpark
+---------------
+
+GeoMesa provides support for working with data stored in Accumulo using the Spark Python API.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+`Spark`_ 2.0 should be installed.
```

Comment:
[picky style] Can we make these bullets to match the other prereq sections we have.
```
* `Spark`_ 2.0 installed.
* `Python`_ 2.7 or 3.x installed.
* `pip`_ or ``pip3``  installed.
```